SpiceSharp/Components/Semiconductors/Bipolar/TransientBehavior.cs

```diff
@@ -569,8 +576,8 @@ public void CalculateExcessPhase(object sender, ExcessPhaseEventArgs args)
             var cbe = args.ExcessPhaseCurrent;
             var gbe = args.ExcessPhaseConduct;
 
-            var delta = StateExcessPhaseCurrentBc.Timesteps[0];
-            var prevdelta = StateExcessPhaseCurrentBc.Timesteps[1];
+            var delta = _method.GetTimestep(0);
+            var prevdelta = _method.GetTimestep(1);
```

Comment:
@svenboulanger Why delta is "0" and prevdelta is "1" ?  It doesn't sounds clear to me.

Reply:
The index is 'how many steps to go back in time`. So an index of 0 would be the *current* timestep, while 1 is the *previous* index. Hence the naming.